### DISCUSSION TITLE
Developer chat

### DISCUSSION THREAD

**Discussion by piiq** [OpenBB-finance/OpenBB#333] [General]
Hey there 🖖
Is there a developer community chat/discord server? I'm looking for a place where I can interact with other other people who are using or building on top of the terminal.

**Comment by DidierRLopes** [comment]
Hey @piiq 

This is an excellent idea! We've just created one: https://discord.gg/FmKjHKMn

Now we need to update the README.md with it and share it with more users.

I'll soon be doing a new post on Reddit and Hacker News to thank everyone for the support so far, and will add the link to the discord server so that more people can know about it
